bundles/org.openhab.extensionservice.marketplace.automation/pom.xml

```diff
@@ -11,7 +11,7 @@
 
   <artifactId>org.openhab.extensionservice.marketplace.automation</artifactId>
 
-  <name>openHAB Add-ons :: Extension Service :: Eclipse IoT Marketplace (Automation)</name>
+  <name>openHAB Add-ons :: Bundles :: Extension Service :: Eclipse IoT Marketplace (Automation)</name>
```

Comment:
Could you please remove all the changes in pom.xml of other bundles from this PR?

Reply:
That's weird. These changes are actually not from me, but are from the last commit (a248d5d30d5a0230eefaa42bd5e7544e212471ba) *before* I forked the repo. However, I saw this automatic error message here ""Signed-off-by" needed on" and then I executed a command to add the signed-off statement to all my previous commits on the forked repo. It seems that unfortunately, the signed-off was also added to the last commit from the openhab repo which was not from me. Not sure if the changes will go away if I just revert the comment.

Reply:
Could you try to rebase the branch then? Btw, I see that you created the PR from your master branch - please better always work on a dedicated feature branch, that will make your life easier, if you ever work on multiple PRs at the same time.

Reply:
Yes, I agree, I'll use a branch instead of a master next time. I was now able to remove my signed-off  from commit 8d246199736567d4a3ab938eeae40d71e1d1214d. However, I now see all commits in this PR twice. I'm not super experienced with git and it's also my first PR and don't want to make it worse. I guess I can't rebase because I'm already on the master? The only clean option I could think of is to re-fork openhab2-addons again, create a branch and merge my changes to that new branch. However, I think I cannot change the source of this PR which is currently the master branch. Sorry for the muddle.

Reply:
I did a rebase so the changes are gone now.